UV_excess_in_low_mass_stars/UV_excess_in_low_mass_stars.ipynb

```diff
@@ -0,0 +1,948 @@
+{
```

Comment:
<p>In retrospect, should loading the catalog of TWA star locations come first, and then search VO? Because that would be the scientific workflow... I've got a big list of stars, I want to know which ones have GALEX data. You could still demonstrate how to search VO with just the one star at first, and then do the loop for the whole list of stars.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>